src/cli/java/org/commcare/util/screen/EntityListSubscreen.java

```diff
@@ -48,71 +53,41 @@ public EntityListSubscreen(Detail shortDetail, Vector<TreeReference> references,
         this.handleCaseIndex = handleCaseIndex;
         references.copyInto(entitiesRefs);
         actions = shortDetail.getCustomActions(context);
-        rows = getRows(entitiesRefs, context, shortDetail);
+        initEntities(context, shortDetail);
```

Comment:
nit: It would make this easier to read if it didn't modify the class but instead returned the list:
```java
entities = initEntities(context, shortDetail)
```

Reply:
resolved in later commit